pearl/utils/functional_utils/train_and_eval/online_learning.py

```diff
@@ -116,7 +116,7 @@ def online_learning(
         if number_of_steps is not None and episode_total_steps > record_period:
             print(
                 f"An episode is longer than the report_period: episode length {episode_total_steps}"
```

Comment:
`report_period` -> `record_period`?

Reply:
Good catch

Reply:
You prefer a new PR?

Reply:
Updated main branch with fix for report_period -> record_period